src/Tribe/RSVP.php

```diff
@@ -1557,7 +1558,7 @@ public function get_event_reports_link( $event_id ) {
 	 *
 	 * @return string
 	 */
-	public function get_ticket_reports_link( $event_id, $ticket_id ) {
+	public function get_ticket_reports_link( $ticket_id ) {
```

Comment:
We shouldn't abruptly change the method signature. Instead, keep `$event_id` and deprecate the parameter. Change the variable name to `$event_id_deprecated`

```php
if ( ! empty( $event_id_deprecated ) ) {
    _deprecated_argument( __METHOD__, 'TBD' );
}
```